website/docs/docs/collaborate/access-from-dbt-cloud.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Access dbt Explorer from dbt Cloud features"
+sidebar_label: "Access from dbt Cloud"
+description: "Learn where and how to directly access and interact with dbt Explorer from dbt Cloud features and products."
+---
+
+<p style={{ color: '#808080', fontSize: '1.1em' }}>
+Access dbt Explorer from other features and products inside dbt Cloud, ensuring you have a seamless experience navigating between resources and lineage in your project. 
+</p>
+
+This guide explains how to access dbt Explorer from various dbt Cloud features, including the dbt Cloud IDE, jobs, and other areas. While the primary way to access dbt Explorer is through the **Explore** link in the navigation, you can also access it from other dbt Cloud features.
```

Comment:
Suggest that we retain use of the word `guide` as a descriptor for the /Guides
```suggestion
This article explains how to access dbt Explorer from various dbt Cloud features, including the dbt Cloud IDE and jobs. While the primary way to navigate to dbt Explorer is through the **Explore** link in the navigation, you can also access it from other dbt Cloud features.
```

Reply:
Good point! I changed it to page?